react/features/chat/components/web/ChatInput.js

```diff
@@ -118,7 +126,9 @@ class ChatInput extends Component<Props, State> {
                                 text = ':)' />
                         </div>
                     </div>
-                    <div className = { smileysPanelClassName }>
+                    <div
+                        className = { smileysPanelClassName }
+                        style = { smileysPanelWidth }>
```

Comment:
@mike-gusev-ae  are you shure? looks like now `smileysPanel` size is equal to the `window.innerWidth` regarding to the line 230. Not sure, didn't run that code. Sorry if I'm wrong.